Client/game.js

```diff
@@ -488,26 +488,24 @@ function likeImage(imageId) {
 	let im = document.getElementById(imageId);
 	if (im.src.includes('like_off.png')) {
 		im.src = 'like_on.png';
-		document.getElementById(imageId.replace('like', 'counter')).style.color = 'green';
+		document.getElementById(imageId.replace('like', 'counter')).className = 'like-counter-on';
 		socket.emit('like image', gameId, imageId);
 	} else if (im.src.includes('like_on.png')) {
 		im.src = 'like_off.png';
-		document.getElementById(imageId.replace('like', 'counter')).style.color = 'black';
+		document.getElementById(imageId.replace('like', 'counter')).className = 'like-counter-off';
```

Comment:
Probably would be better to use Element.classList.remove() and Element.classList.add() ([here are the docs](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList)) in case we want to put more than one class on these elements.

In that vein, it'd make sense to change `.like-counter-off` to just `.like-counter`, give both elements the `.like-counter` class in the HTML, then add/remove the `.like-counter-on` class here.